src/xinterpreter.cpp

```diff
@@ -208,10 +206,22 @@ SEXP try_parse(const std::string& code, int execution_counter) {
 
     void interpreter::configure_impl()
     {
```

Comment:
This is a bit over the top, but the idea here is to figure out where `xr` is and then find the `../share/jupyter/kernels/xr/R/setup.R` file and `source()` it. 

Are there better ways to host extra files ?